### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by Bifabig**
- Add a column species of type string to animals table.

transaction-2-rollback

- Inside a transaction:

   - Update the animals table by setting the species column to unspecified.

transaction-1

   - Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.

transaction-3

   - Update the animals table by setting the species column to pokemon for all animals that don't have species already set.

transaction-3-2

   - Commit the transaction and verify that changes persist after commit.

transaction-3-after commit

- Inside a transaction delete all records in the animals table, then roll back the transaction.

transaction-4

transaction-4-rollback

- Inside a transaction:
   - Delete all animals born after Jan 1st, 2022.

transaction-5

   - Create a savepoint for the transaction.

transaction-5-savepoint

   - Update all animals' weight to be their weight multiplied by -1.

transaction-5-multiplyby-1

   - Rollback to the savepoint

transaction-5-rollbacktosavepoint

   - Update all animals' weights that are negative to be their weight multiplied by -1.

transaction-5-allnegativetopositive

   - Commit transaction

transaction-5-commit

- How many animals are there?
 
transaction-6-count-animals

- How many animals have never tried to escape?

image

- What is the average weight of animals?

transaction-6-avg-weight

- Who escapes the most, neutered or not neutered animals?

transaction-6-never-attempt-escape

- What is the minimum and maximum weight of each type of animal?

transaction-6-min-max-weight

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

transaction-6-last

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,89 @@ SELECT name, date_of_birth from animals WHERE name IN  ('Agumon','Pikachu');
...
+COMMIT;
+
+-- How many animals are there?
+SELECT COUNT(id)
+FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(id)
+FROM animals
+WHERE escape_attempts = 0;
```
Comment (Olamarx) [inline queries.sql:74]: - [ ] I could not see this query, and the one I saw in the place does not give the correct result to it. I will encourage you to rerun it.

image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Bifabig, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Readme file professional ✔️
- Data added ✔️
- Schema added ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Bifabig) [reply queries.sql:74]: Hi @Olamarx,

Thanks for the comments. Screenshots have been corrected!

**Review by amejid — APPROVED** [review]
Hi @Bifabig, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- SQL organized in multiple files :heavy_check_mark: 
- Screenshot provided :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**